docs/Contribute.md

```diff
@@ -0,0 +1,126 @@
+
+First off, here are useful links:
+
+* Cromwell documentation: [cromwell.readthedocs.io/en/develop](http://cromwell.readthedocs.io/en/develop)
+* Source on Github: [github.com/broadinstitute/cromwell](https://github.com/broadinstitute/cromwell/tree/develop/docs)
+* Builds on ReadTheDocs: [readthedocs.org/projects/cromwell](https://readthedocs.org/projects/cromwell/builds/)
+* How to build and view the documentation locally: [mkdocs.readthedocs.io](https://mkdocs.readthedocs.io/en/stable/#installation)
+
+### "needs docs"
+
+There are plenty of areas of the Cromwell documentation that need to be updated, improved, or added. Within the Cromwell repo on Github there are many issues labeled as ["needs docs"](https://github.com/broadinstitute/cromwell/issues?q=is%3Aopen+is%3Aissue+label%3A%22needs+docs%22), so feel free to start there. 
+
+If you would like to request additional documentation, you can create a [Github issue in the Cromwell repo](https://github.com/broadinstitute/cromwell/issues/new).
+
+### Writing Tips
+
+1. Keep it clear, accurate, and concise.
+2. Put the most important information first.
+3. Use the second person, use “you” instead of “the user”.
+4. No passive verbs (everything is done by something).
+5. Link to the original source, don't repeat documentation. 
+
+### Formatting
+
+The documentation is written in Markdown. Click here for a [Github Guide on Markdown](https://guides.github.com/features/mastering-markdown/), and click here for more [tips from MkDocs](http://www.mkdocs.org/user-guide/writing-your-docs/).
+
+### Styling
+
+**Links:**
+
+* Absolute: `[link text](www.destinationURL.com)`  
+_Example:_ `[Broad Institute](www.broadinstitute.org)` _produces this link_ [Broad Institute](www.broadinstitute.org).
+* Relative: `[link text](Destination_Page)`, where `Destination_Page` is the file name without the `.md` extension  
+_Example:_ `[How to use the Cromwell CLI](CommandLine)` _produces this link_ [How to use the Cromwell CLI](CommandLine).
+* Anchor link: `[anchor text](../Path/To/Page#Anchor)`  
+_Example:_ `[HPC filesystems](backends/HPC#filesystems)` _produces this link_ [HPC filesystems](backends/HPC#filesystems).
+
+**Code:**
+
+* To style a word of code, use a backtick (\`) before and after the word.  
+_Example:_ \`file.json\` _produces_ `file.json`. 
+* To style a block of code, use three backticks (\`\`\`) before and after the block of code.  
+_Example:_  
+	\`\`\`  
+		workflow myWorkflow {  
+		call myTask  
+		}  
+	\`\`\`  
+_produces this block_  
+```  
+	workflow myWorkflow {  
+	call myTask  
+	}  
+```  
+* To use syntax highlighting, include the language after the first three backticks (\`\`\`).  
+_Example:_   
+	\`\`\`json  
+		\{  
+		    "MyWorkflow.MyTask.VariableTwo": "Variable2"  
+		\}  
+	\`\`\`	
+_produces this block_  
+```json  
+	{
+	    "MyWorkflow.MyTask.VariableTwo": "Variable2"
+	}
+```	
+
+**REST API:**
```

Comment:
If you do keep the `Styling` section, these `REST API` instructions don't belong in the middle of it